.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -16,7 +16,13 @@ labels: bug
 * then second step
 * I was expecting X but instead it shows Y
 
-Did you try to reproduce this issue at http://cdn.clappr.io/
+**Is this only happenning in your streaming?**
+* If yes, please provide to us the public video URL
```

Comment:
`If it does, please provide your streaming URL.`

Reply:
@leandromoreira Thank you for the syntax improvements hints.